libs/dal/src/repositories/message/message.schema.ts

```diff
@@ -130,6 +134,13 @@ messageSchema.virtual('template', {
   justOne: true,
 });
 
+messageSchema.virtual('actorSubscriber', {
```

Comment:
```suggestion
messageSchema.virtual('actor', {
```

Should we maybe just call it "actor" or there might some reason for this?

Reply:
we already have an existing `actor` field on the message entity with the shape

```js
actor: { 
type: ActorTypeEnum, // USER, SYSTEM ICON, SYSTEM CUSTOM URL
data: string | null
}
```